### PR TITLE
Bump dependency -> Bump dependency in whole project

### PR DESCRIPTION
**Pull request by Nataniel-Antosik**

### CODE REVIEW

**Comment by marcinWisnia** [comment]
@Nataniel-Antosik what do You think about adding https://github.com/renovatebot/renovate this nice, automated dependency tool to Your project :) ?

**Comment by marcinWisnia** [comment]
migrating to https://developer.android.com/build/migrate-to-catalogs would be nice :) You can add it to Your schedule/calendar of improvements :D

**Review by marcinWisnia — APPROVED** [review]

File: app/build.gradle [hunk 1]
```diff
@@ -1,10 +1,13 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 plugins {
     id 'com.android.application'
     id 'org.jetbrains.kotlin.android'
     id 'kotlin-kapt'
     id 'dagger.hilt.android.plugin'
     id 'androidx.navigation.safeargs.kotlin'
     id 'com.google.android.gms.oss-licenses-plugin'
+    id 'com.google.devtools.ksp'
```
Comment (BorysChajdas) [inline app/build.gradle]: Why it is added during the dependencies bump :D?

Comment (Nataniel-Antosik) [reply app/build.gradle]: Because of it 😄
image

**Comment by Nataniel-Antosik** [comment]
@marcinWisnia 
I created issue #179 for `Migrate your build to version catalogs`
I don't know if I would like automatically bump my dependency. I have one example of `oss-license` which I didn't want to bump the version automatically. Maybe I can omit doing it for this library I will look at it in this issue #180

File: app/build.gradle (hunk 1 above)
Comment (Nataniel-Antosik) [reply app/build.gradle]: If you think it is a good idea I can split this PR to add KSP to the project and bump dependency

**Comment by marcinWisnia** [comment]
@Nataniel-Antosik  nice :) according to automatic updating -> this tool is creating a PR, You don't have to merge it :D You can also probably configure it in way You love <3

File: app/build.gradle (hunk 1 above)
Comment (BorysChajdas) [reply app/build.gradle]: Yes, it would be nice to split it :)

**Review by BorysChajdas — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (Nataniel-Antosik) [reply app/build.gradle]: Done 😄

**Pull request merged**